pkg/state/state.go

```diff
@@ -3737,6 +3763,16 @@ func (st *HelmState) getOCIChart(release *ReleaseSpec, tempDir string, helm helm
 	return &chartPath, nil
 }
 
+// IsOCIChart returns true if the chart is an OCI chart
+func (st *HelmState) IsOCIChart(chart string) bool {
+	if strings.HasPrefix(chart, "oci://") {
+		return true
+	}
+
+	repo, _ := st.GetRepositoryAndNameFromChartName(chart)
+	return repo != nil
```

Comment:
Can we safely remove these lines?

```suggestion
```

Reply:
@mumoshu some OCI chart is not start with oci://, but it's repo is oci. WDYT?

Reply:
Ahh good catch!
Then perhaps we shall add another test case like `{&HelmState{}, "ocirepo/mychart:1.0.0", true},`?

Reply:
@mumoshu got it.

Reply:
Thanks for confirming!

Also, we should use something other than `GetRepositoryAndNameFromChartName`, or modify it to see if the repo is oci=true or not, and `IsOCIChart` returning true when `repo != nil && repoIsOCI` 🤔 

Otherwise `IsOCIChart` can return `true` if it's not OCI, which may cause regression to non-OCI chart users.